RHEL6_7/system/tuned/solution.txt

```diff
@@ -1,4 +1,4 @@
-The module stores the custom 'tuned' profiles, which are located in the /etc/tune-profiles/ directory, to POSTUPGRADE_DIR.
-
-After the upgrade to Red Hat Enterprise Linux 7, the profiles can be migrated manually with the help of the 'tuned-man' and 'tuned-profiles' commands to the /etc/tuned/ directory.
+After the upgrade to Red Hat Enterprise Linux (RHEL) 7, the profiles can be migrated manually with the help of the 'tuned-man' and 'tuned-profiles' commands to the /etc/tuned/ directory.
```

Comment:
It's `tuned` and `tuned-profiles` manual pages not commands. Also Tuned documentation (https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html-single/performance_tuning_guide/index#chap-Red_Hat_Enterprise_Linux-Performance_Tuning_Guide-Tuned) could be mentioned.